src/renderers/imgui_dx11/buffers.rs

```diff
@@ -22,26 +20,35 @@ struct VertexConstantBuffer([[f32; 4]; 4]);
 
 impl Buffers {
     pub(crate) fn new(dasc: &DeviceAndSwapChain) -> Self {
-        let mtx_buffer = unsafe {
+        let mut mtx_buffer = None;
+
+        unsafe {
             dasc.dev()
                 .CreateBuffer(
                     &D3D11_BUFFER_DESC {
                         ByteWidth: std::mem::size_of::<VertexConstantBuffer>() as u32,
                         Usage: D3D11_USAGE_DYNAMIC,
-                        BindFlags: D3D11_BIND_CONSTANT_BUFFER.0,
-                        CPUAccessFlags: D3D11_CPU_ACCESS_WRITE.0,
-                        MiscFlags: 0,
+                        BindFlags: D3D11_BIND_CONSTANT_BUFFER,
+                        CPUAccessFlags: D3D11_CPU_ACCESS_WRITE,
+                        MiscFlags: D3D11_RESOURCE_MISC_FLAG(0),
```

Comment:
I wish that for all of these instances we could just put a big fat `0` in there and have the type system figure it out via the plethora of conversion traits -- `From`, `Deref`, whatever -- which is one of Rust's strong suits.